core/src/main/java/org/testcontainers/containers/Network.java

```diff
@@ -42,10 +37,20 @@ static Network newNetwork() {
         private String driver;
 
         @Singular
-        private Set<Consumer<CreateNetworkCmd>> createNetworkCmdModifiers = new LinkedHashSet<>();
+        private Set<Consumer<CreateNetworkCmd>> createNetworkCmdModifiers;
```

Comment:
Lombok's `@Builder` will initialize it to empty collection if `@Singular` is present